dashboard/.storybook/main.js

```diff
@@ -0,0 +1,24 @@
+const path = require('path')
+
+const toPath = (_path) => path.join(process.cwd(), '..', _path)
+
+module.exports = {
+  stories: ['../**/*.stories.mdx', '../**/*.stories.@(js|jsx|ts|tsx)'],
+  addons: ['@storybook/addon-links', '@storybook/addon-essentials'],
+  features: {
+    previewCsfV3: true,
+  },
```

Comment:
This enables the [Component Story Format 3.0](https://storybook.js.org/blog/component-story-format-3-0/) preview feature. This allows for some neat stuff that we're not using yet.